### PR TITLE
fix: button submit in UiQuestion trigger default submit without prevention it

### PR DESCRIPTION
**Pull request by AlanLes**
As it was triggering the URL change in the application, triggering the page reload.

### CODE REVIEW

**Comment by mateuszkulpa** [comment]
I'm wondering if it should be set via `tag` and `attrs`. 
After these changes, we are forcing developers to use the `form` tag and prevent event even though it is not necessary.

**Review by sonya0504 — DISMISSED** [review]

**Comment by AlanLes** [comment]
> I'm wondering if it should be set via `tag` and `attrs`. After these changes, we are forcing developers to use the `form` tag and prevent event even though it is not necessary.

Yes, we are, as it's the UiQuestion component. It should be the `form`, and since it's the form - we want it to `preventDefault` its' submit event. I think it's a valid approach 🤔  😄

**Comment by mateuszkulpa** [comment]
> > I'm wondering if it should be set via `tag` and `attrs`. After these changes, we are forcing developers to use the `form` tag and prevent event even though it is not necessary.
> 
> Yes, we are, as it's the UiQuestion component. It should be the `form`, and since it's the form - we want it to `preventDefault` its' submit event. I think it's a valid approach 🤔 😄

I get it, but shouldn't we rename it `UiForm` then?

**Review by mateuszkulpa — DISMISSED** [review]

File: src/components/templates/UiQuestion/UiQuestion.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <form class="ui-question">
+  <form class="ui-question" @submit.prevent>
```
Comment (pspaczek) [inline src/components/templates/UiQuestion/UiQuestion.vue:2]:
```suggestion
  <form 
    class="ui-question" 
    @submit.prevent
  >
```

**Pull request merged**